app/models/mixins/miq_web_server_worker_mixin.rb

```diff
@@ -89,8 +93,6 @@ def sync_workers
         end
       end
 
-      modify_apache_ports(ports_hash, self::PROTOCOL) if MiqEnvironment::Command.supports_apache?
-
```

Comment:
@abellotti good question.  The UI shows up to 9.  Although, with advanced settings, you can choose more.  This is why the `maximum_workers_count` is set to 10, so even if you choose more, the system won't let you go beyond that max value.